include/up-cpp/transport/UTransport.h

```diff
@@ -51,9 +48,9 @@ namespace uprotocol::utransport {
                * @return Returns OKSTATUS if the payload has been successfully sent (ACK'ed), otherwise it
                * returns FAILSTATUS with the appropriate failure.
                */
-               virtual UStatus send(const UUri &uri, 
-                                    const UPayload &payload,
-                                    const UAttributes &attributes) = 0;
+               virtual uprotocol::v1::UStatus send(const uprotocol::v1::UUri &uri, 
```

Comment:
According to [the spec](https://github.com/eclipse-uprotocol/up-spec/tree/main/up-l1#21-send), the signature should be:
```
OUT UStatus send(UMessage&)
```

Reply:
Yes, David is working on this